schemas/2015-08-01/Microsoft.Compute.json

```diff
@@ -802,13 +802,21 @@
         "name": {
           "type": "string"
         },
-        "subnet": {
-          "$ref": "#/definitions/id"
-        },
-        "loadBalancerBackendAddressPools": {
-          "type": "array",
-          "items": {
+		"properties" : {
```

Comment:
I just saw the second part of your pull request. Just for testing purposes, try adding "additionalProperties": false line just before the "name" property of the ipConfiguration definition, and then run the tests again. I think you'll find that they fail.